bootloaders/uart/uart-pt.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": [1, 0],
+  "unpartitioned": {
+    "families": ["absolute"],
+    "permissions": {
+      "secure": "rw",
+      "nonsecure": "rw",
+      "bootloader": "rw"
+    }
+  },
+  "partitions": [
+    {
+      "start": "128K",
+      "size": "32K",

```

Comment:
If this is intended as a "generic UART bootloader", should we increase the size of this partition to allow for bigger UF2s?

Reply:
The issue is that this simple bootloader just loads the whole partition onto the other device, so the larger the partition the longer it takes. From a quick play it takes 0.35s for this 32kiB partition, vs 5.6s for a full 512kiB partition (which would be the maximum size, as that fills the RP2350 SRAM)

Reply:
Ah, that makes sense. I guess in _theory_ a more complicated UART bootloader could look at the IMAGE_DEF and only copy the part of the partition that it needed to?

I've not played with partitions on the RP2350 yet, but what happens if you try drag'n'dropping a 128kiB image into a partition that's only 32kiB big?